pkg/virt-handler/vm_test.go

```diff
@@ -2192,23 +2192,23 @@ var _ = Describe("VirtualMachineInstance", func() {
 			Expect(blockMigrate).To(BeTrue())
 			Expect(err).To(Equal(fmt.Errorf("cannot migrate VMI with non-shared HostDisk")))
 		})
-		DescribeTable("when host model labels", func(capabilities *nodelabellerapi.Capabilities) {
+		DescribeTable("with host model", func(hostCpuModel string) {
 			vmi := api2.NewMinimalVMI("testvmi")
 			vmi.Spec.Domain.CPU = &v1.CPU{Model: v1.CPUModeHostModel}
 
-			controller.capabilities = capabilities
+			controller.hostCpuModel = hostCpuModel
 
 			err := controller.isHostModelMigratable(vmi)
 
-			if capabilities != nil {
-				Expect(err).ShouldNot(HaveOccurred())
-			} else {
+			if hostCpuModel == "" {
 				Expect(err).Should(HaveOccurred())
+			} else {
+				Expect(err).ShouldNot(HaveOccurred())
 			}
 
 		},
-			Entry("exist migration should succeed", &nodelabellerapi.Capabilities{Host: nodelabellerapi.Host{UUID: "hostUUID"}}),
-			Entry("don't exist migration should fail", nil),
+			Entry("exist migration should succeed", "Westmere"),
+			Entry("don't exist migration should fail", ""),
```

Comment:
As said above I would keep the previous test and expect it to fail, but I won't insist on that.
Other than that looks great.

Awesome work, thanks!
/lgtm